README.md

```diff
@@ -20,7 +20,7 @@ cordova plugin add cordova-plugin-mparticle
 **Install the SDK** using CocoaPods:
 
 ```bash
-$ # Update your Podfile to depend on 'mParticle-Apple-SDK' version 7.2.0 or later
+$ # Update your Podfile to depend on 'mParticle-Apple-SDK' version 8.5.0 or later
```

Comment:
Given the iOS bug, this should say 8.5.2 not 8.5.0, since 8.5.0 has the bug we fixed.

Reply:
oops I had the number for Android in my head. Good catch!

Also you are correct they determine the sdk version for iOS in their podfile